inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -146,6 +149,11 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
     float threshold;
     InferenceEngine::CNNNetwork cnnNetwork;
     std::shared_ptr<InferenceEngine::Core> core;
+    std::vector<ngraph::PartialShape> inputDynamicShape;
+    std::vector<ngraph::Shape> targetStaticShape;
+    std::vector<std::vector<ngraph::Shape>> targetStaticShapes;
```

Comment:
again, `targetStaticShape` and `targetStaticShapes` are confusing. Looks like `targetStaticShape` is a special case of `targetStaticShapes`. It would be good to remove

Reply:
Yes, the targetStaticShape is one of targetStaticShapes, and it is planned to support multiple target static shapes under one input dynamic shape and this is requested by Maxim. And do you mean this needs not to be supported as well?